.github/workflows/scripts/destroy-clusters.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
```

Comment:
wouldn't it have been easier to turn https://github.com/camunda/camunda-tf-rosa/tree/main/.github/workflows/scripts into a reusable action for this case?
Otherwise you we keep copying the file and having to maintain it in two places.